frontend/routes/teamspaces/components/modelDialog/modelDialog.component.tsx

```diff
@@ -52,6 +52,8 @@ interface IProps {
 	createModel: (teamspace, data) => void;
 }
 
+const getModelCodeFieldErrorMsg = (error) => error ? 'Model code can only contain alphanumeric' : '';
```

Comment:
Can we rename this to `Can only contain alphanumeric characters`?

Reply:
> Can we rename this to `Can only contain alphanumeric characters`?

@carmenfan Done. And as we are using it in more than one place I've moved this function to model helpers file.